src/main/haskell/kore/src/Kore/Builtin/String.hs

```diff
@@ -183,14 +222,177 @@ asPartialExpandedPattern
 asPartialExpandedPattern resultSort =
     maybe ExpandedPattern.bottom (asExpandedPattern resultSort)
 
+ltKeyT :: Text
```

Comment:
`builtinFunctions :: Map Text Builtin.Function` and `Builtin.wrongArity`, as well as `[Int/String/Etc].expectBuiltingDomain*` take the same thing as `String`. I was going for integrating this ASAP to avoid from keeping people blocked. I could add a new card to change the `String`s to `Text` and remove the duplication. What do you think?